server/auth/create_token.go

```diff
@@ -0,0 +1,96 @@
+package auth
+
+import (
+	"encoding/json"
+	"github.com/gmo-personal/coding_challenge/server/csrf"
+	"github.com/gmo-personal/coding_challenge/server/utils"
+	"time"
+
+	"github.com/gmo-personal/coding_challenge/database"
+	"github.com/gmo-personal/coding_challenge/model"
+	"golang.org/x/crypto/bcrypt"
+	"net/http"
+)
+
+// Structure of a session, containing account ID and expiry time.
+type Session struct {
+	AccountID string
+	Expires   time.Time
+}
+
+// In memory sessions table to keep track of active sessions.
+// TODO This should be a database table to avoid memory and concurrency issues as well as scaling.
+var Sessions = make(map[string]Session)
```

Comment:
same as with the database - avoid global vars and pass around sessions storage as a value

Reply:
Will fix